### PR TITLE
Hybris 12.1

### PR DESCRIPTION
**Pull request by KonstaT**

### CODE REVIEW

**Comment by KonstaT** [comment]
New branch with static stable/cm-12.1-YOG3C based manifest here.
https://github.com/mer-hybris-kis3/android/tree/hybris-12.1-YOG3C

If wanted to do it right, mer-hybris forks should also have new updated hybris-12.1-YOG3C branches. Anyway, the mer-hybris forks are currently much closer to YOG3C than what you get when you sync the 'rolling' hybris-12.1 manifest today.

File: default.xml [hunk 1]
```diff
@@ -22,7 +22,7 @@
   <project path="hybris/mer-kernel-check" name="mer-hybris/mer-kernel-check" revision="master" />
   <project path="external/droidmedia" name="sailfishos/droidmedia" revision="master" />
 
-  <project path="android" name="CyanogenMod/android" />
+  <project path="android" name="mer-hybris/android" revision="hybris-12.1" />
```
Comment (sledges) [inline default.xml]: Other (older) branches don't sync the manifest, should we nuke it here too? I.e. it's always accessible through .repo/manfests/ anyway

**Comment by sledges** [comment]
I suggest to switch to static branch (as another porter had the same droidmedia issues with rolling cm); please create another PR for that, we'll discuss there

File: default.xml (hunk 1 above)
Comment (KonstaT) [inline default.xml]: Yes, that's fine too.

**Comment by KonstaT** [comment]
Updated. Not sure how to pull request the other branch...

Probably needs these merged and new branch created where I can pull request that additional commit.

**Comment by sledges** [comment]
Create PR like this: https://github.com/mer-hybris/android/compare/hybris-12.1...mer-hybris-kis3:hybris-12.1-YOG3C

This one LGTM :+1:

**Pull request merged**